docs/connect.txt

```diff
@@ -169,6 +169,19 @@ the optional ``error_trace`` argument to ``True``, like so::
 
 .. _authentication:
 
+Backoff Factor
+..............
+
+When attempting to make a request, the connection can be configured so that retries
+are made at lengthening time intervals. This can be configured like so:
+    >>>connection = client.connect(..., backoff_factor=0.1)
+
+The formula for this backoff_factor is:
```

Comment:
I think you can delete this line (and one of the empty new lines). the next para works by itself :)